src/Integration.Vsix.UnitTests/CFamily/CFamilyEmbeddedSonarWayRulesTests.cs

```diff
@@ -44,12 +44,14 @@ public class CFamilyEmbeddedSonarWayRulesTests
         // The QP filter has "active/inactive" tabs. The number of rules is shown in the top-right of the screen.
         // 5. Repeat for C++.
 
-        // Rule data for C-Family plugin v6.41 (build 60884)
-        private const int Active_C_Rules = 209;
-        private const int Inactive_C_Rules = 102;
+        // Rule data for C-Family plugin v6.47.0.62356
 
-        private const int Active_CPP_Rules = 435;
-        private const int Inactive_CPP_Rules = 165;
+        // Note: does not include MISRA rules - see #4604
+        private const int Active_C_Rules = 210;
+        private const int Inactive_C_Rules = 123;
+
+        private const int Active_CPP_Rules = 436;
+        private const int Inactive_CPP_Rules = 207;
```

Comment:
![image](https://github.com/SonarSource/sonarlint-visualstudio/assets/10757559/7bf3ae87-4df4-4501-8d11-cb6febb4661e)


The CFamily plugin defines 436 + 207 = 643 rules.

The "extra" six rules are the now-deprecated "Common" rules, which are all inactive.

![image](https://github.com/SonarSource/sonarlint-visualstudio/assets/10757559/4768ceb9-fe0b-4550-8f36-d93ecf2086f0)

That explains the discrepancy in the inactive rule count: between the two screen shots. 207 inactive C++ + 6 inactive Common = 213 inactive in total.